src/app/cake/cake.model.ts

```diff
@@ -0,0 +1,32 @@
+export interface CakeInterface {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.

Since we are using a Single File Responsibility file distribution, this Interface should live in a separate file named `cake.interface.ts`.